guesscard-index.html

```diff
@@ -12,190 +12,11 @@
-        </div>
-      </div>
-      <div class="cell cell-19">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-20">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-21">
-        <div class="check">
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-22">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-23">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-24">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-25">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-26">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-27">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-28">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-29">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-30">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-31">
-        <div class="check">
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-32">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-33">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-34">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-35">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-      <div class="cell cell-36">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-              
-    </div>
-
+    <!--Div container to hold the grid of 36 cards-->
+    <div class="container" id="container">
```

Comment:
Your browser will understand but please always add an ending tab if necessary.
```suggestion
    <div class="container" id="container"></div>
```

Reply:
OMG, i accidentally deleted while removing the 36 divs :)